client/extensions/woocommerce/state/action-types.js

```diff
@@ -1,3 +1,4 @@
 export const WOOCOMMERCE_EDIT_PRODUCT = 'WOOCOMMERCE_EDIT_PRODUCT';
 export const WOOCOMMERCE_EDIT_PRODUCT_ATTRIBUTE = 'WOOCOMMERCE_EDIT_PRODUCT_ATTRIBUTE';
 export const WOOCOMMERCE_EDIT_PRODUCT_VARIATION = 'WOOCOMMERCE_EDIT_PRODUCT_VARIATION';
+export const WOOCOMMERCE_UPDATE_PRODUCT_VARIATIONS = 'WOOCOMMERCE_UPDATE_PRODUCT_VARIATIONS';
```

Comment:
This should probably be called `WOOCOMMERCE_EDIT_PRODUCT_VARIATIONS` or something to that effect. `UPDATE` doesn't tell me anything about the edit state, so I'd assume it actually performs an API update or something similar.